sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -109,13 +109,18 @@ export class Recorder {
    *
    * Takes SanitizerOptions as the input, passes on to the proxy-tool.
    *
-   * Applied in both record and playback modes.
+   * By default, it applies only to record mode.
    *
-   * (If you want this to be applied only in a specific mode(either "record" or "playback"), check the mode yourself and add the call accordingly.)
+   * If you want this to be applied in a specific mode or in a combination of modes, use the "mode" argument.
    */
-  async addSanitizers(options: SanitizerOptions): Promise<void> {
+  async addSanitizers(
+    options: SanitizerOptions,
+    mode: "record" | "playback" | ["record", "playback"] = "record"
```

Comment:
Nit: I think this may as well just be 
```suggestion
    modes: ("record" | "playback")[]
```

Then the logic below can be simplified to just be `mode.includes(actualTestMode)`.

And an observation -- the current type doesn't won't allow me to specify `["playback", "record"]`. It feels a bit weird that the ordering matters. Super minor issue though.

Reply:
Lol, my initial draft was that and it felt weird to have an array when passing a single mode.

Reply:
Updated as suggested